sklearn/cluster/_bicluster.py

```diff
@@ -558,7 +559,10 @@ def _check_parameters(self, n_samples):
 
     def _fit(self, X):
         n_sv = self.n_components
-        if self.method == "bistochastic":
+        if callable(self.method):
+            normalized_data = self.method(X)
+            n_sv += 1
```

Comment:
If someone add a `log` based normalization then this would be incorrect, isn't it?